microlith/Dockerfile

```diff
@@ -73,6 +52,30 @@ RUN npm install && npm run build -- --base-href=/couchbase/ui/ --deploy-url=/cou
 
 # Couchbase proprietary end
 
+# Generate documentation and include locally as a static site
+# We do not include Antora, just use it: https://gitlab.com/antora/docker-antora/-/blob/main/LICENSE
+FROM antora/antora:3.0.0-alpha.9 as antora-generator
+RUN mkdir -p /antora/cmos
+ENV HOME=/antora
+
+# Antora requires the input be a Git repo with at least one commit: https://gitlab.com/antora/antora/-/issues/284
+# No need to pin
+# hadolint ignore=DL3018
+RUN apk add --upgrade --no-cache git && \
+    git init /antora/docs && \
+    touch /antora/docs/.gitignore && \
+    git -C /antora/docs add . && \
+    git config --global user.name "ignore.me" && \
+    git config --global user.email "ignore.me" && \
+    git -C /antora/docs commit -m "ignore me"
+
+# Put the changeable stuff here
+COPY docs/ /antora/docs/
+# Couchbase proprietary start
+COPY --from=builder /src/github.com/couchbaselabs/cbmultimanager/doc/*.adoc /antora/docs/
```

Comment:
Ah, TIL.

I'll change this to follow the Antora structure - it worked locally, but I agree, best stick to how it expects it - most likely put it in its own Antora module to prevent collisions